### PR TITLE
[#88] Added logging around potential null pointers

### LINKED ISSUE #88: Connector throws `NullPointerException` when the message in exception is empty
In integration test runs, it has been observed a few times, that an exception is thrown which has no error message and this may lead to a `NullPointerException` in `YugabyteDBErrorHandler`.

It can be prevented by adding a check in `YugabyteDBErrorHandler` to not proceed ahead if the error message is found to be null/empty.

### PR DESCRIPTION
**Pull request by vaibhav-yb**
This PR closes #88

### CODE REVIEW

**Review by suranjan — COMMENTED** [review]
I would like to see in which case we get the msg null for debugging purposes. Can we print that exception as well?

**Comment by vaibhav-yb** [comment]
> I would like to see in which case we get the msg null for debugging purposes. Can we print that exception as well?

@suranjan I have added an info log which will print the exception in all the cases so whenever we get this null log, we can simply see the log line just before that and analyse it for debugging purposes.

File: src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java [hunk 1]
```diff
@@ -16,14 +18,22 @@
...
     public YugabyteDBErrorHandler(YugabyteDBConnectorConfig connectorConfig, ChangeEventQueue<?> queue) {
         super(YugabyteDBConnector.class, connectorConfig, queue);
     }
 
     @Override
     protected boolean isRetriable(Throwable throwable) {
+        LOGGER.info("Checking if a retry is possible for retriable exception: {}", throwable);
+
+        if (throwable.getMessage() == null) {
+            LOGGER.warn("Exception message received is null");
```
Comment (suranjan) [inline src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java:32]: We should log the throwable if it is not being logged.

**Review by suranjan — CHANGES_REQUESTED** [review]

**Review by vrajat — APPROVED** [review]

File: src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java (hunk 1 above)
Comment (vaibhav-yb) [reply src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java:32]: It is being logged in line number 29.

**Pull request merged**